packages/adminrouter/extra/src/test-harness/modules/mocker/common.py

```diff
@@ -127,6 +127,9 @@ def _create_common_endpoints():
             ReflectingUnixSocketEndpoint('/run/dcos/dcos-diagnostics.sock'))
         # DC/OS Metronome
         res.append(ReflectingTcpIpEndpoint(ip='127.0.0.1', port=9000))
+        # Checks API
```

Comment:
Can we add corresponding unit test to `adminrouter` test suite?

Reply:
https://github.com/dcos/dcos/pull/3037/commits/917e94d2b2d9d10db0fea0f05d5b1afe8b67e673

Reply:
Thanks @mhrabovcin for keeping an eye on the tests!